### ISSUE TITLE
on members listing, display borough & party

### ISSUE THREAD

**Issue by davidmooreppf** [datamade/nyc-councilmatic#87]
I have taken technical steps to export this official council [page](http://council.nyc.gov/html/members/members.shtml) into a [structured Google sheet database](https://docs.google.com/spreadsheets/d/1v4qc1NlEyMH2abHuy_cKV3i7yv9nvuJoYimdSGc1ti4/edit#gid=0). But I think you have it in OCD already. Just in case helpful. 

Goal is to display both on /council_members, say to right of name & district number? E.g. :: 

Margaret Chin - District 1 - Manhattan - Democrat 

And an enhancement coming next, that would capture the full bio from individual official council member pages' on .gov.

**Comment by fgregg** [comment]
It's a little tricky to represent the borough in a portable way. Would it be okay to show the district on boundaries on a map. We have this now?

**Comment by davidmooreppf** [comment]
I think it's useful, on [Council Members](http://nyc-councilmatic.datamade.us/council-members/) listing, to enable visitors to skim down the page and see info they understand. Currently only displaying the name & district number isn't very accessible. 

Adding party & district to /council-members seems to me like basic info a visitor would expect to see there. 

The real goal in my mind is adding intelligible information there, as people know the city by neighborhood or area name - e.g. "Fort Greene", or a looser "West Side Manhattan", etc. That's also a fast way for new site visitors to find their member without going through an address lookup. But that's not in the scope of this issue, just mentioning it.

**Comment by fgregg** [comment]
So you are saying that a map of the district is not an acceptable
substitute for the name of the borough?

On Sat, Sep 26, 2015 at 5:00 PM, David Moore notifications@github.com
wrote:

> I think it's useful, on Council Members
> http://nyc-councilmatic.datamade.us/council-members/ listing, to enable
> visitors to skim down the page and see info they understand. Currently only
> displaying the name & district number isn't very accessible.
> 
> Adding party & district to /council-members seems to me like basic info a
> visitor would expect to see there.
> 
> The real goal in my mind is adding intelligible information there, as
> people know the city by neighborhood or area name - e.g. "Fort Greene", or
> a looser "West Side Manhattan", etc. That's also a fast way for new site
> visitors to find their member without going through an address lookup.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/datamade/nyc-councilmatic/issues/87#issuecomment-143499330
> .

## 

773.888.2718

**Comment by derekeder** [comment]
@fgregg are we launching with a map?

**Comment by davidmooreppf** [comment]
That's correct, I think members listing could use skimmable info, for now at least in the form of borough and party (eventually, neighborhoods). 

I don't think a map is priority, b/c to me, on a member profile page, it initially looks like a blob. Takes some zooming to understand, but then it's very useful for understanding a district.

**Comment by fgregg** [comment]
@derekeder, I assume that this discussion is not about launch features.

On Sat, Sep 26, 2015 at 5:28 PM, David Moore notifications@github.com
wrote:

> That's correct, I think members listing could use skimmable info, for now
> at least in the form of borough and party (eventually, neighborhoods).
> 
> I don't think a map is priority, b/c to me, on a member profile page, it
> initially looks like a blob. Takes some zooming to understand, but then
> it's very useful for understanding a district.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/datamade/nyc-councilmatic/issues/87#issuecomment-143500650
> .

## 

773.888.2718
